datacommons/test/node_test.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest import mock
+
+import datacommons
+
+
+class TestProperties(unittest.TestCase):
+
+    @mock.patch("datacommons.node._post")
+    def test_with_data(self, _post):
+
+        def side_effect(path, data):
+            if path == "/v1/bulk/properties/out" and data == {
+                    "nodes": ["City"]
+            }:
+                return {
+                    "data": [{
+                        "node":
+                            "City",
+                        "properties": [
+                            "name", "provenance", "subClassOf", "typeOf"
+                        ]
+                    }]
+                }
+
+        _post.side_effect = side_effect
+        response = datacommons.properties(["City"])
+        assert response == {
+            "City": ["name", "provenance", "subClassOf", "typeOf"]
+        }
+
+
+class TestPropertyValues(unittest.TestCase):
+
+    @mock.patch("datacommons.node._post")
+    def test_with_data(self, _post):
```

Comment:
can we also add a test with multiple values for a property?

Reply:
Done